### ISSUE TITLE
Assertions without error messages

### ISSUE THREAD

**Issue by ivanov-v-v** [raphael-group/chisel#7]
Dear Simone,

thank you once again for your package! It works on some of our datasets and does so decently. Unfortunately, on other datasets, it doesn't work, and I am currently debugging the code to identify the underlying issues. Currently, the pipeline fails on "selecting ploidies" step during the calling phase, on some of the assertions. And it is really tricky to figure out the actual problem because in the multiprocessing context the error reduces to

![image](https://user-images.githubusercontent.com/8493747/69906101-86881280-13be-11ea-9f3e-3880c69ab476.png)

It would have been so much better if you could annotate the assertion statements.
I tried to do that myself but the code in `Caller.py` was hard to follow. As this is only a preliminary release, I hope that all these suggestions will contribute to the tool development and better user experience in the future. When I find the actual bug, if there is one, I will also report it

**Comment by simozacca** [comment]
Some assertions are indeed missing a corresponding message and we are in process of fixing it. Thank you for the observation. I will keep the issue open until we add a message to each assertion.

**Comment by simozacca** [comment]
Assertion messages have been added in the last version
